src/client/RosettaRestClientFactory.ts

```diff
@@ -13,7 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import fetch from 'cross-fetch';
```

Comment:
Would this work on browser and node environments? Is fetch out of the box for node js 18+?

Reply:
exactly. so the browsers got support for the fetch api and then someone wrote the `cross-fetch` polyfill, and now node js 18+ supports the api out of the box.

Reply:
there is more information in the release announcement about this:

- https://nodejs.org/de/blog/announcements/v18-release-announce/

Reply:
Awesome, reading https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API#browser_compatibility and other posts

Reply:
People using old node version could provide a custom fetch implementation